charts/opentelemetry-demo-datadog/README.md

```diff
@@ -17,7 +17,7 @@ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm
 
 To install the chart with the release name my-otel-demo, run the following command:
 
-We can install open telemetry collector either as gateway or daemonset. Please create API key from datadog .
+We can install open telemetry collector either as deployment or daemonset. Please create API key from datadog .
```

Comment:
Why change 'gateway' to 'deployment' (it does seem better to me)?

Reply:
To be consistent with naming conventions of operator :)